.github/workflows/codeql-codescan.yml

```
@@ -0,0 +1,44 @@
+  push:
+    branches:
+      - '**'   
+jobs:
+  CodeQL:
+    if: ${{ github.actor != 'dependabot[bot]' }}
+    name: Analyse Code With CodeQL
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'javascript' ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)